packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -599,6 +598,7 @@ export class ServiceQueryCreatorStore extends QueryEditorStore {
 
     // leverage initialization of query builder state to ensure we handle unsupported queries
     queryBuilderState.initializeWithQuery(service.execution.func);
+    await queryBuilderState.changeDetectionState.start();
```

Comment:
I was thinking to call `start` in `initailizeWithQuery` because I am calling `start` everywhere we do that. @akphi what do you think?